### ISSUE TITLE
Screen reader toolbar opened after using Muon Analysis

### ISSUE THREAD

**Issue by DavidFair** [mantidproject/mantid#17554]
### Expected behavior

The toolbars the user had open before using the Muon Analysis window should be reopened when the interface is closed
### Actual behavior

A black toolbar appears which is the screen reader.
### Steps to reproduce the behavior

Open the Muon Analysis Interface
Ensure the option to hide toolbars is selected
Close the interface - Observe a black toolbar has now appeared
### Platforms affected

Windows 7/10 - Possibly All

**Comment by tom-perkins** [comment]
Related: #11693

**Comment by DavidFair** [comment]
Closing as obsolete
